Big2/App.js

```diff
@@ -10,8 +11,13 @@ const App = () => {
     <NavigationContainer>
       <Stack.Navigator
         initialRouteName="Home"
-        screenOptions={{ gestureEnabled: false }}
+        rulesRoute="Rules"
```

Comment:
This line seems unnecessary to create a route to 'Rules', can you explain this one to me?